diff --git a/node_modules/@strapi/admin/admin/src/pages/HomePage/CloudBox.js b/node_modules/@strapi/admin/admin/src/pages/HomePage/CloudBox.js
index f67b7a0..0d04f70 100644
--- a/node_modules/@strapi/admin/admin/src/pages/HomePage/CloudBox.js
+++ b/node_modules/@strapi/admin/admin/src/pages/HomePage/CloudBox.js
@@ -11,6 +11,7 @@ import cloudIcon from './assets/strapi-cloud-icon.svg';
 
 const BlockLink = styled.a`
   text-decoration: none;
+  display: none;
 `;
 
 const CloudCustomWrapper = styled(Box)`
diff --git a/node_modules/@strapi/admin/admin/src/pages/HomePage/ContentBlocks.js b/node_modules/@strapi/admin/admin/src/pages/HomePage/ContentBlocks.js
index 84ec0ac..3829b48 100644
--- a/node_modules/@strapi/admin/admin/src/pages/HomePage/ContentBlocks.js
+++ b/node_modules/@strapi/admin/admin/src/pages/HomePage/ContentBlocks.js
@@ -10,6 +10,7 @@ import CloudBox from './CloudBox';
 
 const BlockLink = styled.a`
   text-decoration: none;
+  display: none;
 `;
 
 const ContentBlocks = () => {
diff --git a/node_modules/@strapi/admin/admin/src/pages/HomePage/index.js b/node_modules/@strapi/admin/admin/src/pages/HomePage/index.js
index 2d44595..43b6fb6 100644
--- a/node_modules/@strapi/admin/admin/src/pages/HomePage/index.js
+++ b/node_modules/@strapi/admin/admin/src/pages/HomePage/index.js
@@ -78,14 +78,7 @@ const HomePage = () => {
               />
             </GridItem>
           </Grid>
-          <Grid gap={6}>
-            <GridItem col={8} s={12}>
-              {showGuidedTour ? <GuidedTourHomepage /> : <ContentBlocks />}
-            </GridItem>
-            <GridItem col={4} s={12}>
-              <SocialLinks />
-            </GridItem>
-          </Grid>
+          
         </Box>
       </Main>
     </Layout>
